modules/stanford_profile_styles/stanford_profile_styles.libraries.yml

```diff
@@ -1,3 +1,15 @@
+admin:
+  version: VERSION
+  css:
+    theme:
+      dist/css/admin.css: {}
+
+paragraph:
+  version: VERSION
+  css:
+    theme:
+      dist/css/paragraph.css: {}
```

Comment:
can we rename this to something like `stanford_profile_styles.field_widgets.css`

Reply:
👍

Reply:
@pookmish,
Does this mean you want the filename on the file system to be
`dist/css/stanford_profile_styles.field_widgets.css` ?

Reply:
Yes. it essentially namespaces the files with their source. and the same for the `admin.css`